clj-refactor.el

```diff
@@ -573,7 +573,7 @@ list of (fn args) to pass to `apply''"
 
 (defun cljr--whitespacep (s)
   "True if S contains only whitespace."
-  (s-blank? (string-trim s)))
```

Comment:
essential difference between `s-blank?` and `string-blank-p` that former is null safe latter is not. I don't think that causes any problem in this commit but you might want to dblcheck that..